style.css

```diff
@@ -308,6 +308,98 @@ a {
   background: #fff;
 }
 
+/* Contact section styles */
+
+.contact-form-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.contact-text {
+  padding: 20px;
+  margin: 10px 5px;
+}
+
+.contact-text-h3 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.shimmer {
+  transition: transform 2s;
+}
+
+.shimmer:hover {
+  transform: translate(10px);
+}
+
+.contact-form {
+  margin: 10px 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+fieldset {
+  display: flex;
+  flex-direction: column;
+  border: none;
+}
+
+.contact-form input {
+  margin: 10px 5px;
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  box-sizing: border-box;
+  border-radius: 4px;
+  min-width: 327px;
+  height: 48px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #6b778c;
+  padding: 15px 16px;
+}
+
+.contact-form textarea {
+  margin: 10px 5px;
+  background: #fff;
+  color: #091e42;
+  border: 1px solid #36b37f;
+  padding: 12px;
+  border-radius: 4px;
+  min-width: 327px;
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 500;
+  width: 99%;
+}
```

Comment:
- [ ] Kindly adjust your width on the form `textarea` because it have take more than 100% width of the screen :wink:

   - #### Your design
   ![image](https://user-images.githubusercontent.com/87197412/187409033-7db2f8b6-8bdb-484f-8709-b78e78fc7b5f.png)


   - ### Figma design
    ![image](https://user-images.githubusercontent.com/87197412/187409652-4f9d86b9-e108-42a2-b4f6-cc37b269da62.png)